chapters/synchronous.tex

```diff
@@ -295,8 +295,8 @@ \subsection{Argument Restrictions (Component Expression)}\label{argument-restric
 \end{lstlisting}
 \end{example}
 
-Note that the operators defined in this chapter do not automatically vectorize,
-but some operate on arrays in a similar way.
+None of the operators defined in this chapter vectorize, but some can pass through an array of clocked expressions (but not of clocks).
```

Comment:
I'll try to be more constructive by showing what I think was meeting's intention (as formulated by @sjoelund):
```suggestion
None of the operators defined in this chapter vectorize, but some can operate directly on array variables (including clocked array variables, but not clock array variables).
```